methode-content-collection-mapper.yaml

```diff
@@ -101,4 +101,8 @@ logging:
     io.dropwizard: DEBUG
   appenders:
     - type: console
-      logFormat: "%-5p [%d{ISO8601, GMT}] %c: %X{transaction_id} %m [%thread]%n%xEx"
\ No newline at end of file
+      logFormat: "%-5p [%d{ISO8601, GMT}] %c: %X{transaction_id} %m [%thread]%n%xEx"
+
+appInfo:
+  systemCode: "MethodeContentCollectionMapperApplication"
+  description: "MethodeContentCollectionMapperApplication"
```

Comment:
Please add a proper (short) description of the service.
This is from Dewey, I would shorten this a bit:
"Methode Content Collection Mapper (MCCM) is responsible for listening to new content collection publishes (i.e. content package or story package) on the kafka NativeCMSPublicationEvents queue, and then transforming them into UPP format and then writing them to kafka CMSPublicationEvents queue."

Reply:
done